arc/data/ear-claims-key-attestation-ko.json

```diff
@@ -0,0 +1,22 @@
+{
+  "eat_profile": "tag:github.com,2023:veraison/ear",
+  "iat": 1666529184,
+  "ear.verifier-id": {
+    "developer": "https://veraison-project.org",
+    "build": "vts 0.0.1"
+  },
+  "ear.raw-evidence": "NzQ3MjY5NzM2NTYzNzQK",
+  "submods": {
+    "PARSEC_TPM": {
```

Comment:
Should be `TPM_PARSEC` to be consistent with `TPM_ENACTTRUST`?

Reply:
See my reply here: https://github.com/thomas-fossati/draft-ear/pull/20#discussion_r1120007128